cmd/namespace-divergence/divergence.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/google/go-github/github"
+	"github.com/nikoksr/notify"
+	"github.com/nikoksr/notify/service/slack"
+	"golang.org/x/oauth2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func main() {
+	// list of excluded Namespaces
+	excludedNamespaces := []string{
+		"cert-manager",
+		"default",
+		"ingress-controllers",
+		"kube-node-lease",
+		"kube-public",
+		"kube-system",
+		"kuberhealthy",
+		"kuberos",
+		"logging",
+		"opa",
+		"overprovision",
+		"velero",
+	}
+
+	// get arguments from the user
+	var kubeconfig *string
+	if home := homedir.HomeDir(); home != "" {
```

Comment:
What is the idea behind setting "" if home directory is empty. Can we get it from the env var KUBECONFIG